src/SignalR/samples/SignalRSamples/Startup.cs

```diff
@@ -25,15 +94,22 @@ public class Startup
         // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddHttpContextAccessor();
             services.AddConnections();
 
             services.AddSignalR(options =>
             {
-                // Faster pings for testing
-                options.KeepAliveInterval = TimeSpan.FromSeconds(5);
+                options.AddFilter<CustomHubFilter>();
+                options.AddFilter(new InstanceFilter());
+            })
+            .AddHubOptions<Chat>(options =>
+            {
+                options.AddFilter(new InstanceFilter());
```

Comment:
Should it be possible to remove global filters on a per hub basis?

Reply:
That's our existing pattern here https://github.com/dotnet/aspnetcore/blob/fc313376dd74861e422298d81482dcd86e73a8da/src/SignalR/server/Core/src/HubOptionsSetup%60T.cs#L12.

Seems like we should populate the per hub options with the global filters so users can observe and mutate the list.

Reply:
Do we want to expose the currently internal list? How would people identify the filters so they can choose what to remove? Especially when they pass a `T` or `Type` and we wrap it in a `HubFilterFactory`?

Reply:
Yeah. I'm not so sure about this either. Coming up with a filter descriptor seems like a lot of effort for a niche feature. It could always be added later if it's a real pain point.

Reply:
I'm OK not having this for now I guess. Seems off to not expose a list but lets skip it for now and consider it